ci-operator/config/cri-o/cri-o/cri-o-cri-o-main.yaml

```diff
@@ -68,6 +68,24 @@ tests:
   steps:
     cluster_profile: gcp
     workflow: openshift-e2e-gcp
+- always_run: false
+  as: perfscale-control-plane-6nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "6"
+      ENABLE_LOCAL_INDEX: "true"
+      ES_TYPE: qe
+      EXTRA_FLAGS: --churn-duration=20m --pod-ready-threshold=13s
+      ITERATION_MULTIPLIER_ENV: "15"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-control-plane
+    - chain: openshift-qe-orion-consolidated
+    workflow: openshift-qe-installer-aws
+  timeout: 3h30m0s
```

Comment:
Can you increase this to `8h` - the first attempt failed due to the `3h30m` https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/59005/rehearse-59005-pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes/1859057021169438720#1:build-log.txt%3A488

Regarding the versioning, can we confirm `cri-o://1.32.0-1.ci.el9` is the expected build? if so, lgtm!

Reply:
>Regarding the versioning, can we confirm cri-o://1.32.0-1.ci.el9 is the expected build?

Yup, that's expected. We recently merged a change that helps us identify the CRI-O binary built with the PR content.

Reply:
@jtaleric did you see the recent failure? https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/59005/rehearse-59005-pull-ci-cri-o-cri-o-main-perfscale-control-plane-6nodes/1859242723970650112

It looks like CI couldn't find the `rhel-coreos-base` image.

Reply:
I am not sure why one run would succeed but following ones fail? Could be a build server thing?

Reply:
It appears your new attempt got past that issue @sohankunkerkar